core/domain/exp_services_test.py

```diff
@@ -1059,21 +1535,21 @@ def test_get_image_filenames_from_exploration(self):
                     '<oppia-noninteractive-image filepath-with-value='
                     '"&amp;quot;s3Choice1.png&amp;quot;" caption-with-value='
                     '"&amp;quot;&amp;quot;" alt-with-value='
-                    '"&amp;quot;&amp;quot;"></oppia-noninteractive-image>'
+                    '"&amp;quot;&amp;quot;" image_id-with-value="4"></oppia-noninteractive-image>'
                     '</p>'),
                 (
                     '<p>This is value2 for ItemSelection'
                     '<oppia-noninteractive-image filepath-with-value='
```

Comment:
Ditto. (here any elsewhere in this file)

Reply:
done